### ISSUE TITLE
Kindly, would you please remove this repository?

### ISSUE THREAD

**Issue by Gregg** [mahmoud-eskandari/vuemastery-dl#80]
Gregg Pollack here, founder of Vue Mastery.

I love putting out content-free, so it can be accessible to everyone. That's why just this last weekend we unlocked all of our content so everyone in the world (not just those who can afford it) can get access to the learning.  Last year because of covid, we put out the courses free for a whole week (to do our part to help).  Earlier this year we also made individual courses free, and we will continue to do that.

I'm sad to see that a library like this exists to help people download our videos illegally.  The revenue from Vue Mastery supports the livelihood for a group of workers (contractors and employees) and we give 25% of every dollar we get paid back to Vue.js and all the core contributors who dedicate their time to supporting it.  We give out a 50% discount to just about everyone who asks.. if they're a student or in a country where purchasing parity is an issue.

We're doing our hardest to do good in the world, so a part of me feels sad when I see things like this library.  

Would you kindly remove it?

**Comment by mahmoud-eskandari** [comment]
Yes of course, This was a personal project in my Github account, I had to make this project public because of last year GitHub sanctions against Iranian developers (We could not have private projects).
We can't even pay for accounts because of international banking sanctions.

I was also surprised by the much attention to this repository.
Please read my answer and send me a reply to be sure to read my answer.

Thank you, I apologize to you

**Comment by mahmoud-eskandari** [comment]
Also, using the Vimeo`s player  is a bug because of It does not check any authentication.

**Comment by mahmoud-eskandari** [comment]
I waiting for your reaction to remove it ASAP
